tests/test_item.py

```diff
@@ -20,6 +22,46 @@ def get_example_item_dict(self) -> Dict[str, Any]:
             item_dict: Dict[str, Any] = json.load(f)
         return item_dict
 
+    def test_utc_datetimes_are_not_localized(self) -> None:
+        item_dict = self.get_example_item_dict()
+
+        prev_tz = os.environ.get("TZ")
+
+        # Test without TZ environment variable set
+        with self.subTest(tz=None):
+            if "TZ" in os.environ:
+                del os.environ["TZ"]
+                time.tzset()
+            item = Item.from_dict(item_dict)
+            datetime = item.datetime
+            assert datetime is not None
+            self.assertIs(datetime.tzinfo, tz.tzutc())
+            self.assertIsNot(datetime.tzinfo, tz.tzlocal())
+
+        # Test with TZ environment variable set to UTC
+        with self.subTest(tz="UTC"):
+            os.environ["TZ"] = "UTC"
+            time.tzset()
+            item = Item.from_dict(item_dict)
+            datetime = item.datetime
+            assert datetime is not None
+            self.assertIs(datetime.tzinfo, tz.tzutc())
+            self.assertIsNot(datetime.tzinfo, tz.tzlocal())
+
+        # Test with TZ environment variable set to UTC
```

Comment:
```suggestion
        # Test with TZ environment variable set to US/Central
```

Or just remove the comments, it's clear (to me at least) what's going on here.

Reply:
Removed in 6997000e40f248ea8779f53d5b41c3f535aa9a51